Automattic/jetpack
# Show social review prompt with both Social and Jetpack installed

## Proposed changes:

* This diff allows the Jetpack Social review prompt to show if the main Jetpack plugin is active. Previously, if you were using the Jetpack Social plugin and also had the main Jetpack plugin active, the review request prompt would not show
* This is a follow-up to #27917 

### Other information:


## Jetpack product discussion


pbNhbs-5DY-p2#comment-13811

## Does this pull request change what data or activity we track or use?


* No, it just shows the review prompt in more cases. There are some analytics events associated with the review prompt which will now be recorded in more cases.

## Testing instructions:






* Follow the testing plan on #27917
* Note that if you are testing with the Jetpack Beta plugin, you will need to set the Jetpack plugin to use the Release Candidate instead of the latest stable.
* Test with both Jetpack Social and the main Jetpack plugin active and confirm that you still receive the review prompt when the conditions in the test plan are met.
